releasenotes/notes/update-magnum-containers-8814b1425ec45fc0.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Updates Magnum to a proper Caracal release version following critical bug
```

Comment:
Current version will also bump magnum-capi-helm to latest pypi version i.e. 1.2.0